lib/full-site-editing/class-wp-rest-templates-controller.php

```diff
@@ -168,7 +168,11 @@ public function get_item_permissions_check( $request ) {
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function get_item( $request ) {
-		$template = gutenberg_get_block_template( $request['id'], $this->post_type );
+		if ( isset( $request['is_custom'] ) && 'false' === $request['is_custom'] ) {
+			$template = gutenberg_get_block_file_template( $request['id'], $this->post_type );
```

Comment:
`gutenberg_get_block_template` already falls back to `gutenberg_get_block_file_template` so we don't need this if block? (we can just leave it as is)

Reply:
The purpose of this `if` clause is that we can request the theme file even if we have a database entry for the template.